### PR TITLE
Feature/modem dfu

### PR DESCRIPTION
**Pull request by chunfantasy**
This PR is due to support modem dfu.

### CODE REVIEW

File: lib/actions/modemTargetActions.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+};
+
+
+export const cancelUpdate = () => dispatch => {
+    dispatch(targetActions.modemWritingCloseAction());
+};
+
+export const performUpdate = () => (dispatch, getState) => {
+    dispatch(targetActions.modemWritingStartAction());
+    dispatch(targetActions.modemProcessUpdateAction('Modem DFU just starts.'));
```
Comment (bihanssen) [inline lib/actions/modemTargetActions.js]: Consider re-writing the text message, IMO it's unclear what it means.

File: lib/components/ModemUpdateDialogView.jsx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+                        <div>{ modemFwName }</div>
+                    </FormGroup>
+                    <FormGroup>
+                        <ControlLabel>Process</ControlLabel>
+                        <div>{ progressMsg }</div>
+                    </FormGroup>
+                    <FormGroup>
+                        {isWritingSucceed &&
+                            <Alert bsStyle="success">
+                                Complete  successfully in {this.state.timer} seconds.
```
Comment (bihanssen) [inline lib/components/ModemUpdateDialogView.jsx]: Should be s/complete/completed/ ?
And seems to have one space too many.

File: lib/actions/modemTargetActions.js (hunk 1 above)
Comment (chunfantasy) [reply lib/actions/modemTargetActions.js]: Not started and starting work very nice!

File: lib/components/ModemUpdateDialogView.jsx (hunk 2 above)
Comment (chunfantasy) [reply lib/components/ModemUpdateDialogView.jsx]: Fixed.

**Pull request merged**